### PR TITLE
Show beta info in header

### PR DESCRIPTION
**Pull request by annavik**
Fixes https://github.com/RolnickLab/ami-platform/issues/336

Was it something like this you had in mind @mihow ?

Screenshot 2024-01-31 at 21 36 23

Screenshot 2024-01-31 at 21 36 28

### Comments:
- Do we want to show some more info on hover? Feel free to tweak the copy!
- Version I pick from package.json (FE project), is this how we should define the the "app version"?
- Got kind of crowded in header on small screens, I skipped the info pages on small screens for now.

### CODE REVIEW

**Comment by mihow** [comment]
Amazing, thank you! Perhaps for the tooltip text: "All data is considered test data and may change or be deleted at any time. Use with caution."

**Comment by annavik** [comment]
> Amazing, thank you! Perhaps for the tooltip text: "All data is considered test data and may change or be deleted at any time. Use with caution."

Yes! Copy is updated and version changed to latest commit hash:

Screenshot 2024-01-31 at 22 27 54

**Comment by mihow** [comment]
I think BETA looks better as in your first! We can save Preview Release when we make a release!
And a slight tweak on copy. 
"All data is considered test data and may be changed or deleted at any time. Use with caution."

I can make these changes too if you are done for the night!

**Comment by annavik** [comment]
> All data is considered test data and may be changed or deleted at any time. Use with caution.

No worries, copy tweaked! :)

File: ui/vite.config.ts [hunk 1]
```diff
@@ -1,9 +1,14 @@
 import react from '@vitejs/plugin-react'
+import childProcees from 'child_process'
 import { defineConfig } from 'vite'
 import eslint from 'vite-plugin-eslint'
 import svgr from 'vite-plugin-svgr'
 import viteTsconfigPaths from 'vite-tsconfig-paths'
 
+const commitHash = childProcees
```
Comment (mihow) [inline ui/vite.config.ts:8]: That's cool!

**Review by mihow — APPROVED** [review]

**Pull request merged**